.processes/release.md

```diff
@@ -206,8 +206,6 @@ The following are a series of manual tasks that are needed to be executed for th
 
 - [ ] Deploy a set of `$release` CT nodes for our testnet using our **`cover-traffic` deployment script**.
 
-- [ ] Deploy a set of `$release` cloud nodes for our testnet to support DEADR and be used as relayers (to be removed after https://github.com/hoprnet/hoprnet/issues/2537 is fixed).
```

Comment:
its not clear to me whether this is actually fixed?

Reply:
This should now (again) be done automatically by the CD. As it does on `master`.